src/Rendering.cpp

```diff
@@ -45,17 +59,27 @@ void RenderingModule::recreate_render_surface_texture()
 	render_texture = SDL_CreateTexture(state.renderer, SDL_PixelFormatEnum::SDL_PIXELFORMAT_RGBA8888, SDL_TextureAccess::SDL_TEXTUREACCESS_STATIC, w, h);
 	SDL_SetRenderTarget(state.renderer, render_texture);
 
+	if (render_texture2 != nullptr)
+	{
+		SDL_DestroyTexture(render_texture2);
+	}
+	render_texture2 = SDL_CreateTexture(state.renderer, SDL_PixelFormatEnum::SDL_PIXELFORMAT_RGBA8888, SDL_TextureAccess::SDL_TEXTUREACCESS_STREAMING, w, h);
+
 	// render to screen
 	SDL_SetRenderTarget(state.renderer, nullptr);
 	SDL_RenderClear(state.renderer);
 	SDL_RenderCopyEx(state.renderer, render_texture, nullptr, nullptr, 0, nullptr, SDL_FLIP_NONE);
+	SDL_RenderCopyEx(state.renderer, render_texture2, nullptr, nullptr, 0, nullptr, SDL_FLIP_NONE);
 	SDL_RenderPresent(state.renderer);
+
 }
 
 void RenderingModule::process_signal(WindowResizedSignal& signal)
 {
 	auto& state = AccessUnique<WindowingState>::access_unique();
 	SDL_SetWindowSize(state.window, signal.width, signal.height);
+	this->w = signal.width;
```

Comment:
Looking at this, it would be much nicer to have the fields called `width` and `height` or something like that